settings/settings-browser.php

```diff
@@ -30,7 +30,7 @@ public function __construct( &$polylang ) {
 			array(
 				'module'        => 'browser',
 				'title'         => __( 'Detect browser language', 'polylang' ),
-				'description'   => __( 'When the front page is visited, set the language according to the browser preference', 'polylang' ),
+				'description'   => __( 'When the front page is visited, redirects to the homepage in the browser preferred language. As this doesn\'t work if the front page is cached, we will attempt to disable the front page cache for known cache plugins.', 'polylang' ),
```

Comment:
We should probably word it like `When the front page is visited, redirects to the front page...`, since 'front page' and 'home page' may refers to different pages when 'Static pages' are selected in _Settings > Reading_

Reply:
It seems we are all agree with the sentences

> When the front page is visited, redirects to itself in the browser preferred language. As this doesn't work if it is cached, Polylang will attempt to disable the front page cache for known cache plugins.
 
See https://wpsyntex.slack.com/archives/C017GJQBZP1/p1622733983070000?thread_ts=1622551579.029600&cid=C017GJQBZP1